### PR TITLE
gpuscout: new package

### PR DESCRIPTION
**Pull request by stepanvanecek**

### CODE REVIEW

File: var/spack/repos/builtin/packages/gpuscout/package.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        extension="tar.gz",
+    )
+    version("master", branch="master")
+
+    depends_on("cmake@3.27:", type="build")
+    depends_on("cuda@12:")
+
+    def cmake_args(self):
+        args = []
+        return args
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/gpuscout/package.py]: This is not needed:
```suggestion
```

File: var/spack/repos/builtin/packages/gpuscout/package.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Gpuscout(CMakePackage):
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/gpuscout/package.py]: Would this benefit from a `CudaPackage` inheritance too?

**Review by alalazo — DISMISSED** [review]

**Comment by alalazo** [comment]
@alecbcs I didn't noticed this was already assigned, I left the comments above, but feel free to proceed as you like.

**Comment by alecbcs** [comment]
@alalazo no worries! I'd planned to take a look yesterday to try the package out and review, but you beat me to it.

@stepanvanecek ping me once you've addressed the comments from @alalazo.

File: var/spack/repos/builtin/packages/gpuscout/package.py (hunk 2 above)
Comment (stepanvanecek) [reply var/spack/repos/builtin/packages/gpuscout/package.py]: Good suggestion, thanks.

**Review by alalazo — APPROVED** [review]
Thanks, I'll leave to @alecbcs a final review

**Comment by stepanvanecek** [comment]
@alecbcs I pushed in my changes as suggested by @alalazo . Could you please have a look at the PR?

**Comment by alalazo** [comment]
I'll go ahead and merge. @alecbcs Let me know if you have further requests for changes, we can follow up with another PR in case.

**Review by alalazo — APPROVED** [review]

**Pull request merged**